text-serializer-gson/src/main/java/net/kyori/adventure/text/serializer/gson/GsonComponentSerializer.java

```diff
@@ -90,6 +91,15 @@ static Builder builder() {
    */
   @NonNull UnaryOperator<GsonBuilder> populator();
 
+  /**
+   * Deserialize a component to output of type {@link JsonElement}.
+   *
+   * @param component the component
+   * @return the json element
+   * @since 4.7.0
+   */
+  @NonNull JsonElement serializeToJsonElement(final @NonNull Component component);
```

Comment:
this method feels a bit verbose to me -- `serializeToElement` or `serializeToTree` maybe?

Reply:
definitely - that does sound a lot nicer!